lib/streamlit/elements/multiselect.py

```diff
@@ -53,7 +53,7 @@ def multiselect(
 
         Returns
         -------
-        [str]
+        list
```

Comment:
Can we be more specific here? Also, it would be a good idea to add typings to the function.

Reply:
It is difficult to be more specific here since the output of multiselect is a subset of `options` which elements can be of any type. 

The idea of adding typing is a good idea. 
I was not sure how adding typing should affect a documentation string, so I tried to make a minimal possible change to fix the inconsistency.

Reply:
It is difficult to be more specific here since the output of multiselect is a subset of options which elements can be of any type.

The idea of adding typing is a good idea.
I was not sure how adding typing should affect a documentation string, so I tried to make a minimal possible change to fix the inconsistency.